### PR TITLE
updating selected destination path for clip, & refresh for all folders

### PR DESCRIPTION
**Pull request by jackgrossman18**
ref #1639

### CODE REVIEW

File: modules/Hoot/tools/clipDataset.js [hunk 1]
```diff
@@ -193,7 +193,7 @@ export default class ClipDataset {
             let row         = d3.select( `#row-${ mapId }` ),
                 datasetName = row.select( '.datasetName' ),
                 outputName  = row.select( '.outputName' ),
-                folderId    = row.select( '.outputPath').property( '_value' );
+                folderId    = Hoot.folders.findBy( 'path', row.select( '.outputPath' ).property( 'value' ) ).id;
```
Comment (brianhatchl) [inline modules/Hoot/tools/clipDataset.js]: The `_value` property should have the folder id.  Was this change correcting something else?

Comment (jackgrossman18) [reply modules/Hoot/tools/clipDataset.js]: Before this change, you couldn't change the destination path correctly. 

If the folder path provided when the modal became available would be the folder path the clipped dataset was sent to, regardless of what you had selected the path to be.

Comment (brianhatchl) [reply modules/Hoot/tools/clipDataset.js]: Looking elsewhere in the code, I think it should be `.attr('_value')`
that might fix it.

**Review by brianhatchl — CHANGES_REQUESTED** [review]
There is one more to be replaced on line 136.

**Review by brianhatchl — APPROVED** [review]

**Pull request merged**